org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -59,6 +59,10 @@ <h3>Fixed Bugs</h3>
   <li><code>synthetic</code> methods that represent Kotlin <code>suspend</code>
       functions should not be ignored
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/804">#804</a>).</li>
+  <li>Removed misleading parameters "includes" and "excludes" from "dump",
```

Comment:
Isn't this a non-functional change or even a API change?

Reply:
> even a API change?

sounds too strong to me, especially because Maven allows to declare values for non-existing parameter 😉 

> Isn't this a non-functional

isn't this a bug in documentation? :wink:

Reply:
Ok, then we keep it as a bug!